cve_bin_tool/checkers/openswan.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for openswan
+
+https://www.cvedetails.com/product/57217/Xelerance-Openswan.html?vendor_id=20146
+
+"""
+import os
+from ..util import regex_find
+
+
+def get_version(lines, filename):
+    """returns version information for openswan as found in a given file.
+    The version info is returned as a tuple:
+        [modulename, is_or_contains, version]
+
+    modulename will be openswan if openswan is found (and blank otherwise)
+    is_or_contains indicates if the file is a copy of openswan or contains one
+    version gives the actual version number
+
+    VPkg: xelerance, openswan
+    """
+    regex = [r"Openswan ([0-9]+\.[0-9]+\.[0-9]+)"]
+    version_info = dict()
+
+    binary_names = [
+        "klipsdebug",
+        "showhostkey",
+        "ranbits",
+        "eroute",
+        "showpolicy",
+        "spigrp",
+        "pluto",
+        "_pluto_adns",
```

Comment:
Since _copyright turned out to not be a binary file, I suppose I should ask if _pluto_adns is also a non-binary file.

Reply:
Yeah, my bad. I have removed it too, and checked others, the remaining are binaries.

Reply:
Great!  I think we're good to merge this, then.